.travis.yml

```diff
@@ -1,42 +1,33 @@
-dist: trusty
-sudo: false
-language: go
-go:
-- 1.8.x
+####################################################################################
+## NOT FOR UPSTREAM PROPOSAL; INTENDED FOR CI OF AZURE EXAMPLES IN THIS REPO ONLY ##
+####################################################################################
```

Comment:
once these issues have been addressed - is it possible to revert the changes to this file?